package.json

```diff
@@ -4,7 +4,7 @@
   "description": "A javascript library for building applications on top of Gnosis, the Ethereum prediction market platform",
   "scripts": {
     "lint": "eslint ./",
-    "doc": "cd docs; make html",
+    "doc": "cd docs/source/ && sphinx-build . ../build && node -e 'require(\"open\")(\"../build/index.html\")'",
```

Comment:
`make html` would fail with an error about how jsdoc couldn't find the `../../src` folder because it was searching relative to one level deep. Also, the README talks about the `doc` task also opening the docs after the build, so I've added that functionality back.

Reply:
Okay ;)
Then the Makefile is not needed.

Reply:
Ah great, I forgot to ask you about that, so thanks :)